### PR TITLE
Split nodepool role into v2 and v3

### PR DESCRIPTION
**Pull request by jamielennox**
In preperation for zuul v3 allow the playbook to deploy both.

Signed-off-by: Jamie Lennox <jamielennox@gmail.com>

### CODE REVIEW

**Review by ConsoleCatzirl — COMMENTED** [review]
:+1: Looks good to me, but I'll let others review as well

**Review by mlangbehn — COMMENTED** [review]
Looks good to me.

File: roles/nodepool/tasks/nodepool_v3.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- meta: flush_handlers
+
+- name: Enable and start systemd service
+  service:
+    enabled: yes
+    state: running
+    service: "{{ item }}"
+  with_items:
+    - nodepool-builder
+    - nodepoold
```
Comment (gandelman-a) [inline roles/nodepool/tasks/nodepool_v3.yml:36]: It looks like the only thing that differs between nodepool_v2.yml  and nodepool_v3.yml is the services.  Could we have an internal '_nodepool_service_list' var that we use for with_items here, that is conditionally set to the correct services for the version of nodepol we're using? Then only have one nodepool.yml for both files

**Review by gandelman-a — APPROVED** [review]
This looks good. Gonna approve it in and we can start iterating on top of it in hoist proper, instead of our own forks.

**Pull request merged**